src/ray/core_worker/task_interface.cc

```diff
@@ -177,11 +175,10 @@ Status CoreWorkerTaskInterface::SubmitActorTask(ActorHandle &actor_handle,
       ObjectID::FromBinary(actor_handle.ActorID().Binary());
   builder.SetActorTaskSpec(actor_handle.ActorID(), actor_handle.ActorHandleID(),
                            actor_creation_dummy_object_id,
+			   actor_handle.ActorCursor(), // previous_actor_task_dummy_object_id
```

Comment:
```suggestion
			   /*previous_actor_task_dummy_object_id=*/actor_handle.ActorCursor(),
```

Reply:
I like this better in general, but it makes the line too long so the linter complains.

Reply:
Wait, the line is already too long so the linter complains....